playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/node_upgrade.yml

```diff
@@ -0,0 +1,31 @@
+- name: Prepare for Node evacuation
+  command: >
+    {{ openshift.common.admin_binary }} manage-node {{ openshift.common.hostname | lower }} --schedulable=false
+  delegate_to: oo_first_master
+
+- name: Evacuate Node for Kubelet upgrade
+  command: >
+    {{ openshift.common.admin_binary }} manage-node {{ openshift.common.hostname | lower }} --evacuate --force
+  delegate_to: oo_first_master
+
+- include: rpm_upgrade.yml
+  vars:
+     component: "node"
+     openshift_version: "{{ openshift_pkg_version | default('') }}"
+  when: not openshift.common.is_containerized | bool
+
+- include: containerized_upgrade.yml
+  when: openshift.common.is_containerized | bool
+
+# This will restart the node
+- name: Restart openvswitch service
```

Comment:
I'm honestly not sure if it's the tasks name that is wrong or the actual action.  I'll have to investigate.